exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,8 @@
 const name = " Daniel  ";
-
+let message = "My name is " + name + " and my name is 6 characters long."
 console.log(message);
+var nameTrim = name.trim();
```

Comment:
Great work just one thing, it may be better to use let or const instead of var here.

Reply:
> Great work just one thing, it may be better to use let or const instead of var here.

Yes, I agree. Thank you, Hamra